### PR TITLE
Moved: StefansTools.grepWin version 2.0.1098

### PR DESCRIPTION
**Pull request by KaranKad**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44089)

### CODE REVIEW

File: manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: StefansTools.grepWin
+PackageVersion: 2.0.9
```
Comment (ImJoakim) [inline manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.1098
```

File: manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: StefansTools.grepWin
+PackageVersion: 2.0.9
```
Comment (ImJoakim) [inline manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.1098
```

File: manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: StefansTools.grepWin
+PackageVersion: 2.0.9
```
Comment (ImJoakim) [inline manifests/s/StefansTools/grepWin/2.0.9/StefansTools.grepWin.yaml:5]:
```suggestion
PackageVersion: 2.0.1098
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**